test/hw2/test_task1.py

```diff
@@ -43,7 +43,7 @@ def test_get_longest_diverse_words(self):
         assert result_start_with == checking[: len(result_start_with)]
 
     def test_get_rarest_char(self):
-        rarest_char = "z"
+        rarest_char = "B"
```

Comment:
If you changed the test and do not change data, that means that the behavior of the code changed.

What was changed in that commit?